doc/release-notes/5.9-release-notes.md

```diff
@@ -0,0 +1,157 @@
+# Dataverse Software 5.9
+
+This release brings new features, enhancements, and bug fixes to the Dataverse Software. Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights
+
+### Dataverse Collection Optimizations
+
+The Dataverse Collection page, which also serves as the search page and the homepage in most Dataverse installations, has been optimized, with a specific focus on number of queries for each page load.  The optimizations will be more noticable on Dataverse installations with higher traffic.
```

Comment:
"with a specific focus on reducing the number of queries required for each page load. These optimizations..."?

Reply:
good suggestion - added in https://github.com/IQSS/dataverse/pull/8284/commits/b3eb322bee7e10d2852f558b5ea0a5ea9619a439